.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: java
+
+#Travis changed trusty as default distro. Trusty requires sudo with jkd7
+sudo: required

```

Comment:
This didn't help. :/

It seems that Travis is currently broken. Just google and other projects have same issue with jdk7 and trusty. About default travis: https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming

I made another branch from master (before these changes) and it fails also: https://travis-ci.org/Turmio/rpm-maven-plugin/builds/273793846

Actual development was made by mirroring to GitLab and used GitLabCI: https://gitlab.com/Turmio/rpm-maven-plugin/pipelines/ . I just made patch from there and was very surprised to see Travis failing.

Reply:
I think I found reason for this: 
https://github.com/travis-ci/travis-ci/issues/7884

"My guess is that we can no longer support oraclejdk7 on our recent build images due to Oracle's withdrawal. http://www.webupd8.org/2017/06/why-oracle-java-7-and-6-installers-no.html"